bounties/level-2/stellar-lab-albedo.md

```diff
@@ -0,0 +1,37 @@
+# Sign with Albedo on Stellar Laboratory
+
+| Hunter | Agent
+| :- | :-
+| 1500 XLM | 6 Credits
+
+## Description
+
+### What is this task?
+
+People using Stellar Laboratory can only use Ledger, Trezor and Freighter to sign transactions. Unfortunately, these options only work with desktop computers. With majority of global traffic being on mobile ([source](https://www.statista.com/statistics/277125/share-of-website-traffic-coming-from-mobile-devices/)), this possibly hinders adoption of Stellar network to mobile users.
+
+### What are the requirements for the bounty hunter?
+
+The developer should have knowledge of Stellar Laboratory and Stellar SDK.
+
+### What are the deliverables?
+The developer should provide a completed fork of the laboratory implementing new functionality with at least below items:
+ - [ ] Add UI to initiate signing with Albedo
+ - [ ] Integrate with Albedo
+ - [ ] Transaction must be signed successfully
+
+#### *Judging Criteria and Metrics*
+ - [ ] Code Quality - Is the code well-organized and documented? Does it adhere to language-specific style guidelines?
+ - [ ] UI/UX - Is it comprehensive to users? Is it aligned with current design of Laboratory?
+
+#### *Submission Procedure*
+
+ - Fork the laboratory [repository](https://github.com/stellar/laboratory) with implementing the solution
+ - Create a [new issue](https://github.com/tyvdh/stellar-quest-bounties/issues/new) in the bounty [repository](https://github.com/tyvdh/stellar-quest-bounties)
+   - Choose `review <title of the bounty issue>` as the issue's title
+   - In the new issue description link to the original bounty issue and your fork
+   - Add a comment to the bounty issue linking to your newly created review issue to denote it being ready for review
```

Comment:
Also include a requirement for a pending PR in the Lab repo

Reply:
I've added the PR requirement